src/main/java/ai/pluggy/client/PluggyApiService.java

```diff
@@ -1,11 +1,6 @@
 package ai.pluggy.client;
 
-import ai.pluggy.client.request.AccountsRequest;
-import ai.pluggy.client.request.ConnectorsSearchRequest;
-import ai.pluggy.client.request.CreateItemRequest;
-import ai.pluggy.client.request.TransactionsSearchRequest;
-import ai.pluggy.client.request.UpdateItemMfaRequest;
-import ai.pluggy.client.request.UpdateItemRequest;
+import ai.pluggy.client.request.*;
```

Comment:
Maybe we should keep the import for each one?

Reply:
Yes, this went to the refactor of my IDE